terraform.tf

```diff
@@ -95,6 +95,7 @@ module "ecs" {
   aws_elasticsearch_region = "${var.region}"
   aws_elasticsearch_key    = "${module.es.es_user_access_key_id}"
   aws_elasticsearch_secret = "${module.es.es_user_access_key_secret}"
+  redis_url                = "${module.elasticache_redis.endpoint}"
```

Comment:
The `endpoint` is just the hostname so for it to be a URL (which our Redis library needs and is correctly named as the ENV variable) we need to at least attach a protocol when we assign this data to the variable named URL.

I tested successfully with the app using:
``` 
redis_url                = "redis://${module.elasticache_redis.endpoint}"
```

Reply:
This has been updated now 👍